bigtable/Makefile

```diff
@@ -1,4 +1,7 @@
-dev_build_version=$(shell git describe --tags --always --dirty)
+# dev_build_prefixed_version has the format "bigtable/<dev_version>"
+dev_build_prefixed_version=$(shell git describe --tags --always --match 'bigtable/*' --dirty)
+# dev_build_version is like dev_build_prefixed_version, but without the "bigtable/" prefix
+dev_build_version=$(shell echo $(dev_build_prefixed_version) | sed 's/^bigtable\///')
```

Comment:
Tags will be specific for each emulator, for example:
* `bigtable/v1.0.0`
* `bigtable/v1.2.3`
* `storage/v2.0.1`
* etc

So the first thing we need is to filter that tags from this specific emulator, which is what the `--match` option does).

The VERSION that we use for the Docker build, though, must _not_ be prefixed (and that's ok), so we strip it.